5.validation_plate/1.cellprofiler_processing/nbconverted/0.create_loaddata_with_illum_csv.py

```
@@ -0,0 +1,68 @@
+# In[3]:
+
+
+updated_dfs = loaddata_csv.add_illumination_columns_to_loaddata_csvs(
+    loaddata_csv_paths=list(orig_loaddata_csvs.rglob("loaddata_*.csv")),
+    illum_directory=illum_directory,
+    channel_map=CHANNEL_MAP,
+    output_dir=output_dir,
+)
+
+# Print one plate example
+display(next(iter(updated_dfs.values())).head())
```

Ruff (0.16.2)

[error] Undefined name `display`

(F821)